templates/store/reducers/index.ts

```diff
@@ -0,0 +1,40 @@
+import { {{capitalize name}}Actions, {{capitalize name}}ActionTypes } from '../actions';
+
+export interface {{capitalize name}}State {
+  isLoading: boolean;
+  isError: boolean;
+  data: any
+}
+
+const defaultState: {{capitalize name}}State = {
+  isLoading: false,
+  isError: false,
+  data: null,
+};
+
+export const {{capitalize name}}Reducer = (
+  state = defaultState,
+  action: {{capitalize name}}Actions,
+): {{capitalize name}}State => {
+  switch (action.type) {
+    case {{ capitalize name }}ActionTypes.FETCH_{{ toUpperCase name }}:
+      return {
+        ...state,
+        isLoading: true,
```

Comment:
mnaybe here isError should be set to false on each FETCH action.

Reply:
Fixed